Makefile

```diff
@@ -163,7 +163,16 @@ ch_backup/version.txt:
 
 
 .PHONY: install-deps
-install-deps: .install-deps
+install-deps: check-environment .install-deps
+
+.PHONY: check-environment
+check-environment:
+	@if ! command -v ${PYTHON} > /dev/null; then \
```

Comment:
I'm not sure about `/dev/null` in makefile. For example, it is not mounted in chroot created in teamcity by pbuilder for building deb package and we will get an error.

Reply:
Rewrote the check to not use `/dev/null`.